tests/functional/cylc-combination-scripts/02-vr-restart.t

```diff
@@ -39,11 +38,16 @@ export WORKFLOW_RUN_DIR="${RUN_DIR}/${WORKFLOW_NAME}"
 sed -i 's@P1Y@P5Y@' flow.cylc
 run_ok "${TEST_NAME_BASE}-runs" cylc vr "${WORKFLOW_NAME}"
 
+# cat "${TEST_NAME_BASE}-runs.stdout" >&2
+
 # Grep for vr reporting revalidation, reinstallation and playing the workflow.
 grep "\$" "${TEST_NAME_BASE}-runs.stdout" > VIPOUT.txt
 named_grep_ok "${TEST_NAME_BASE}-it-revalidated" "$ cylc validate --against-source" "VIPOUT.txt"
 named_grep_ok "${TEST_NAME_BASE}-it-installed" "$ cylc reinstall" "VIPOUT.txt"
-named_grep_ok "${TEST_NAME_BASE}-it-played" "$ cylc play" "VIPOUT.txt"
+named_grep_ok "${TEST_NAME_BASE}-it-played" "cylc play" "VIPOUT.txt"
+# Ensure that we don't have two copies of the workflow name
+# https://github.com/cylc/cylc-flow/pull/5377
+grep_fail "${WORKFLOW_NAME} ${WORKFLOW_NAME}" VIPOUT.txt
```

Comment:
Not sure there's any use in grepping for this. There are a multitude of ways the command could be wrong

Reply:
This is honestly a fair point.